### PR TITLE
obs-studio: 29.0.2 -> 29.1.3

### PR DESCRIPTION
**Pull request by fpletz**
###### Description of changes

Based on #229700 with the patch from @materusPL.

https://github.com/obsproject/obs-studio/releases/tag/29.1.0
https://github.com/obsproject/obs-studio/releases/tag/29.1.1
https://github.com/obsproject/obs-studio/releases/tag/29.1.2
https://github.com/obsproject/obs-studio/releases/tag/29.1.3

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/video/obs-studio/default.nix [hunk 1]
```diff
@@ -50,20 +53,20 @@ let
...
-    sha256 = "sha256-TIUSjyPEsKRNTSLQXuLJGEgD989hJ5GhOsqJ4nkKVsY=";
+    sha256 = "sha256-D0DPueMtopwz5rLgM8QcPT7DgTKcJKQHnst69EY9V6Q=";
     fetchSubmodules = true;
   };
 
   patches = [
     # Lets obs-browser build against CEF 90.1.0+
     ./Enable-file-access-and-universal-access-for-file-URL.patch
-    ./Provide-runtime-plugin-destination-as-relative-path.patch
+    ./fix-nix-plugin-path.patch
```
Comment (drupol) [inline pkgs/applications/video/obs-studio/default.nix:69]: Could you please include a comment here? https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/missing-patch-comment.md

**Review by drupol — DISMISSED** [review]

**Comment by drupol** [comment]
Please remove it from draft when the change is implemented so I can see it again in my Github PR list. thx!

**Comment by flexiondotorg** [comment]
Thanks for working on the 29.1.3 update @fpletz 

While you're here, could you also add the fix to enable hardware acceleration for browser sources?

 - https://github.com/NixOS/nixpkgs/issues/226468#issuecomment-1512079448

File: pkgs/applications/video/obs-studio/default.nix (hunk 1 above)
Comment (fpletz) [reply pkgs/applications/video/obs-studio/default.nix:69]: I don't agree. The patch is aptly named. No need for further comments.

**Pull request merged**